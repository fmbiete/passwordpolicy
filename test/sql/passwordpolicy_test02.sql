DROP USER IF EXISTS test_pass;

CREATE USER test_pass WITH PASSWORD 'aaaa';

CREATE USER test_pass WITH PASSWORD 'aaaaaaaaaaaaaaa';

CREATE USER test_pass WITH PASSWORD 'aaaaaaaaaaaaaaa1234';

CREATE USER test_pass WITH PASSWORD 'aaaaaaaaaaaaaaa#*#134';

CREATE USER test_pass WITH PASSWORD 'ASWaaaaaaaaasdf#*#134';

DROP USER IF EXISTS test_pass;
