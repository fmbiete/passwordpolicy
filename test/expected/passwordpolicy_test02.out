DROP USER IF EXISTS test_pass;
NOTICE:  role "test_pass" does not exist, skipping
CREATE USER test_pass WITH PASSWORD 'aaaa';
ERROR:  password is too short.
CREATE USER test_pass WITH PASSWORD 'aaaaaaaaaaaaaaa';
ERROR:  password must contain at least 1 numeric characters.
CREATE USER test_pass WITH PASSWORD 'aaaaaaaaaaaaaaa1234';
ERROR:  password must contain at least 1 special characters.
CREATE USER test_pass WITH PASSWORD 'aaaaaaaaaaaaaaa#*#134';
ERROR:  password must contain at least 1 upper case letters.
CREATE USER test_pass WITH PASSWORD 'ASWaaaaaaaaasdf#*#134';
DROP USER IF EXISTS test_pass;
